testing/unit/test_faucetizer.py

```diff
@@ -319,6 +319,123 @@ def test_devices_learned_and_authenticated(self):
         self._verify_behavioral_config(expected_config)
 
 
+class FaucetizerBehaviorWithoutTailACLTestCase(FaucetizerTestBase):
+    """Test Faucetizer's behavior with no tail_acl setting in Forch config"""
+
+    ORCH_CONFIG = """
+    unauthenticated_vlan: 100
+    """
+
+    FAUCET_STRUCTURAL_CONFIG = """
+        dps:
+          t1sw1:
+            dp_id: 111
+            interfaces:
+              1:
+                output_only: true
+              6:
+                stack: {dp: t2sw1, port: 6}
+              7:
+                stack: {dp: t2sw2, port: 7}
+              23:
+                lacp: 3
+          t2sw1:
+            dp_id: 121
+            interfaces:
+              1:
+                description: HOST
+                max_hosts: 1
+              2:
+                description: HOST
+                max_hosts: 1
+              6:
+                stack: {dp: t1sw1, port: 6}
+        """
+
+    FAUCET_BEHAVIORAL_CONFIG = """
+        dps:
+          t1sw1:
+            dp_id: 111
+            interfaces:
+              1:
+                output_only: true
+              6:
+                stack: {dp: t2sw1, port: 6}
+              7:
+                stack: {dp: t2sw2, port: 7}
+              23:
+                lacp: 3
+          t2sw1:
+            dp_id: 121
+            interfaces:
+              1:
+                description: HOST
+                max_hosts: 1
+                native_vlan: 100
```

Comment:
why is 100 here?  Likewise, can this generic config template be promoted to the superclass?